src/toxcore/dht/packet/nodes_response.rs

```diff
@@ -145,10 +142,10 @@ impl ToBytes for NodesResponsePayload {
 impl FromBytes for NodesResponsePayload {
     named!(from_bytes<NodesResponsePayload>, do_parse!(
         nodes_number: le_u8 >>
-        nodes: cond_reduce!(
+        nodes: map_opt!(cond!(
```

Comment:
The same should be done here as for `OnionRequest2`.

Reply:
done